Node/core-DirectLineWebSockets/README.md

```diff
@@ -0,0 +1,181 @@
+# Direct Line Bot Sample (using client WebSockets)
+
+A sample bot and a custom client communicating to each other using the Direct Line API.
```

Comment:
Somewhere near the top of the sample, we should note the following:

* For many developers, the [DirectLineJs ](https://github.com/Microsoft/BotFramework-DirectLineJs) project is an easy way to connect to the Direct Line in Node.js
* If you're interested in writing your own client without using DirectLineJs, use this sample